### PR TITLE
docs(1vs2): add extra reasons, rewrite existing

### PR DESCRIPTION
**Pull request by alexcg1**
@maximilianwerk @JoanFM I still think we need some more meat on these bones, and some extra code snippets would be nice. Thoughts?

### CODE REVIEW

File: .github/2.0/cookbooks/1vs2.md [hunk 1]
```diff
@@ -61,6 +61,29 @@ Benchmark #1: python -c "from jina import Document, Flow, Executor"
...
+<summary>Click to see code</summary>
+
+```python
+from jina import Document
+
+d = Document(content='hello',
+             uri='https://jina.ai',
+             mime_type='text/plain',
+             granularity=1,
+             adjacency=3,
```
Comment (maximilianwerk) [inline .github/2.0/cookbooks/1vs2.md:82]: Remove these three parameters. To complicated.

File: .github/2.0/cookbooks/1vs2.md [hunk 2]
```diff
@@ -61,6 +61,29 @@ Benchmark #1: python -c "from jina import Document, Flow, Executor"
...
-1. Keyword 4
-1. Keyword 5
\ No newline at end of file
+## 3. 2.0 Gives YOU the Power
+
+Jina 1.0 was a complex beast with lots of boilerplate code and not so much transparency. In 2.0 we're putting the power back in your hands. Want to write your own `.logger`, `.save()`, `.load()` or context manager interface for your Executor? Now that power is yours.
+
+## 4. 2.0 is More Pythonic
+
+In Jina 2.0 Documents and DocumentArrays are first-class citizens and fully Pythonic data types. With no more Drivers, you can manipulate Documents directly with all the power that entails:
```
Comment (maximilianwerk) [inline .github/2.0/cookbooks/1vs2.md:70]: Perhaps: 

We embrace explicit above implicit: No more Drivers. You can manipulate Documents in custom Executors directly with all the power that entails:

```python
from jina import Document, DocumentArray, Executor, requests

class MyExecutor(Executor):
    @requests
    def add_world(self, docs: DocumentArray):
        for doc in docs:
            doc.text += 'world'
```

**Review by maximilianwerk — COMMENTED** [review]
Some suggestoins

File: .github/2.0/cookbooks/1vs2.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-   - 1.2x faster on startup;
-   - 1.2x faster on `import`;
-   - 5% faster on empty Flow.
+Learning Jina 2.0 is as simple as 1, 2, 3. With only three main concepts - Document, Flow, and Executor - you'll get up to speed in no time. Know them all, nothing more, and you're good to go!
+
+## 2.0 is Faster
+
+Compared to 1.x (`1.4.2`), 2.0 is 
+- 20% faster on startup;
+- 20% faster on `import`;
```
Comment (hanxiao) [inline .github/2.0/cookbooks/1vs2.md:11]: actually it is 220%, you can calculate by divide it

File: .github/2.0/cookbooks/1vs2.md [hunk 4]
```diff
@@ -1,47 +1,97 @@
...
-   - 1.2x faster on `import`;
-   - 5% faster on empty Flow.
+Learning Jina 2.0 is as simple as 1, 2, 3. With only three main concepts - Document, Flow, and Executor - you'll get up to speed in no time. Know them all, nothing more, and you're good to go!
+
+![](../flow-executor-doc.svg)
+
+## 2.0 is Faster
+
+Compared to 1.x (`1.4.2`), 2.0 is 
+- 20% faster on startup;
```
Comment (hanxiao) [inline .github/2.0/cookbooks/1vs2.md:12]: 220%

**Comment by alexcg1** [comment]
@hanxiao @maximilianwerk @JoanFM any additional feedback? I've added some images, reformatted performance benchmark and included Max's rewrite.

File: .github/2.0/cookbooks/1vs2.md [hunk 5]
```diff
@@ -51,16 +101,65 @@ Benchmark #1: python -c "from jina import Document, Flow, Executor"
...
+
 </details>
 
-1. Keyword 3
-1. Keyword 4
-1. Keyword 5
\ No newline at end of file
+## 2.0 Gives YOU the Power
+
+Jina 1.0 was a complex beast with lots of boilerplate code and not so much transparency. In 2.0 we're putting the power back in your hands. Want to write your own `.logger`, `.save()`, `.load()` or context manager interface for your Executor? Now that power is yours.
```
Comment (JoanFM) [inline .github/2.0/cookbooks/1vs2.md:143]: the context manager interface I am not sure is accurate. I think we already expose __enter__ but u can redefine __exit__? Have to check code again

File: .github/2.0/cookbooks/1vs2.md [hunk 6]
```diff
@@ -51,16 +101,65 @@ Benchmark #1: python -c "from jina import Document, Flow, Executor"
...
+class MyExecutor(Executor):
+    @requests
+    def add_world(self, docs: DocumentArray):
+        for doc in docs:
+            doc.text += 'world'
+```
+
+## 2.0 is Lighter
+
+Jina has gone on a diet for 2.0. We've dramatically cut the layers of abstraction, removing Drivers, Peas and Pods to give you a simpler, more elegant Flow experience.
```
Comment (JoanFM) [inline .github/2.0/cookbooks/1vs2.md:161]: removing Drivers and hiding Peas and Pods may be better?

Comment (maximilianwerk) [reply .github/2.0/cookbooks/1vs2.md:161]: Yes, this should definitely be changed.

Comment (alexcg1) [reply .github/2.0/cookbooks/1vs2.md]: I worded it as "de-emphasizing Peas and Pods". We talk a lot about being more transparent elsewhere, so "hiding" goes against that

File: .github/2.0/cookbooks/1vs2.md (hunk 5 above)
Comment (alexcg1) [reply .github/2.0/cookbooks/1vs2.md]: I removed it for now

File: .github/2.0/cookbooks/1vs2.md (hunk 4 above)
Comment (alexcg1) [reply .github/2.0/cookbooks/1vs2.md]: Fixed

**Comment by alexcg1** [comment]
Feedback implemented. Are we good to go? If so I'll post to Slack tomorrow with Han's nice image he created. (Since many of our users are in Europe and it's EOD here now)

**Comment by alexcg1** [comment]
You can view it with the embedded images (yoinked from @maximilianwerk 's EAH deck) [here](https://github.com/jina-ai/jina/blob/docs-1vs2-complete/.github/2.0/cookbooks/1vs2.md)

**Pull request merged**